README.md

```diff
@@ -24,11 +24,11 @@ The primary features of `cmp` are:
   by using an `Ignore` option (see `cmpopts.IgnoreUnexported`) or explicitly
   compared using the `AllowUnexported` option.
 
-See the [GoDoc documentation][godoc] for more information.
+See the [GoDev documentation][godev] for more information.
```

Comment:
See the [documentation][...] for more information.

Reply:
Done.